lifecycle_test.go

```diff
@@ -456,6 +456,83 @@ func TestMergePortBindings(t *testing.T) {
 	}
 }
 
+func TestPortMappingCheck(t *testing.T) {
+	makePortMap := func(ports []string) nat.PortMap {
```

Comment:
suggestion: changing to a variadic makes calling cleaner
```suggestion
	makePortMap := func(ports ...string) nat.PortMap {
```

Reply:
https://github.com/testcontainers/testcontainers-go/pull/2658/commits/5fe2f77b1c6101b2c550f182ec4021d828ff5bd0